Makefile

```diff
@@ -1,8 +1,4 @@
-prefix ?= /usr/local
-exec_prefix ?= $(prefix)
-bindir ?= $(exec_prefix)/bin
-datarootdir ?= $(prefix)/share
-docdir ?= $(datarootdir)/doc/displayplacer
+bindir ?= /usr/local/bin
```

Comment:
It wasn't boilerplate. `prefix` and `exec_prefix` are standard autotools variables users may wish to override. And your `install` target should install your LICENSE file.